docs/resources/specification.md

```diff
@@ -98,13 +100,11 @@ It...
 
 When reading from the [EventStore](#EventStore) an `EventStream` is returned.
 
-It...
-
-- It _MUST_ be iterable
+- It _MUST_ be iterable (e.g. via generator or reactive pattern, depending on the specific implementation)
```

Comment:
Still very confusing to me

Reply:
it means, that the event stream can be consumed one event at the time.
do you have a better suggestion?